### PR TITLE
Rfa01a sidenav implementation cals 5362

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### Which User story does this relate to?
-  https://osi-cwds.atlassian.net/browse/CALS-5362
https://osi-cwds.atlassian.net/browse/CALS-4888

#### Brief feature description
- implement side navigation for rfa 01a 

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- npm run rebuild-assets

#### Code coverage percent diff as a result of this PR?
-81.02% Statements
67.46% Branches 
76.54% Functions 
81.96% Lines

### CODE REVIEW

File: app/javascript/rfa_forms/rfa01a_edit_view/a01SideBar.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import {NavLinks, NavLink} from 'react-wood-duck'
+
+const A01SideBar = ({
+  isNavLinkActive,
+  handleNavLinkClick,
+  activeNavLinkHref,
+  hideRelationshipBetweenApplicants
+}) => {
+  return (
+    <div className='col-lg-12 hide-mobile'>
```
Comment (nasar-cambria) [inline app/javascript/rfa_forms/rfa01a_edit_view/a01SideBar.js]: Are we getting any styles from this attribute ```hide-mobile``` ?

**Comment by lauherkcambria** [comment]
@nasar-cambria as according to our convo with leon, i will do a quick investiation if the collapsible library has any default chevrons for us to use. As the desgin for the chevrons have not been decided, a follow up story will be created and added to backlog for once design is complete.

File: app/javascript/rfa_forms/rfa01a_edit_view/a01SideBar.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                  clickHandler={() => handleNavLinkClick('#applicants-card')}
+                  text='1. Applicant Information'
+                  href='#applicants-card' />
+                <NavLink
+                  active={isNavLinkActive('#applicant-residence-card')}
+                  clickHandler={() => handleNavLinkClick('#applicant-residence-card')}
+                  text='2. Applicant Residence'
+                  href='#applicant-residence-card' />
+                {
+                  hideRelationshipBetweenApplicants === 'hidden' ? null
```
Comment (Nasir2712) [inline app/javascript/rfa_forms/rfa01a_edit_view/a01SideBar.js]: good catch man!

File: app/javascript/rfa_forms/rfa01a_edit_view/a01SideBar.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      <div className='col-lg-2'>
+        <Affix>
+          <hr className='vertical' />
+        </Affix>
+      </div>
+    </div>
+  )
+}
+
+A01SideBar.defaultProps = {
```
Comment (Nasir2712) [inline app/javascript/rfa_forms/rfa01a_edit_view/a01SideBar.js]: can you add proptypes here?

Comment (lauherkcambria) [reply app/javascript/rfa_forms/rfa01a_edit_view/a01SideBar.js]: good catch :)

**Pull request merged**